app/src/main/java/com/breadwallet/tools/util/PushNotificationService.kt

```diff
@@ -0,0 +1,34 @@
+package com.breadwallet.tools.util
+import android.R
+import android.app.Notification
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Intent
+import android.media.RingtoneManager
+import android.util.Log
+import androidx.core.app.NotificationCompat
+import com.google.android.gms.tasks.OnCompleteListener
+import com.google.firebase.messaging.FirebaseMessaging
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+
+
+class PushNotificationService : FirebaseMessagingService() {
```

Comment:
last time I already remove this and change to https://github.com/litecoin-foundation/litewallet-android/blob/develop/app/src/main/java/com/litewallet/notification/LitewalletMessagingService.kt